Demo/Demo/Extensions/CustomTextField.swift

```diff
@@ -0,0 +1,16 @@
+import UIKit
+
+class CustomTextField: UITextField {
+    convenience init(placeholderText: String? = nil, defaultValue: String? = nil) {
+        self.init(frame: .zero)
+
+        translatesAutoresizingMaskIntoConstraints = false
+        borderStyle = .roundedRect
+        placeholder = placeholderText
+        text = defaultValue
+        clearButtonMode = .never
+        if let textDelegate = self as? UITextFieldDelegate {
+            delegate = textDelegate
+        }
+    }
```

Comment:
Same comment in here on custom init in extension

Reply:
See https://github.com/paypal/iOS-SDK/pull/14#discussion_r722604924